winfetch.ps1

```diff
@@ -572,14 +572,92 @@ function info_uptime {
 
 # ===== RESOLUTION =====
 function info_resolution {
-    Add-Type -AssemblyName System.Windows.Forms
-    $displays = foreach ($monitor in [System.Windows.Forms.Screen]::AllScreens) {
-        "$($monitor.Bounds.Size.Width)x$($monitor.Bounds.Size.Height)"
+    Add-Type -AssemblyName System.Core
+    Add-Type -AssemblyName System.Runtime.InteropServices
+    $id = Get-Random
```

Comment:
Why do we need a random number for class invocation?

Reply:
When I was using `Add-Type -TypeDefinition`, the type can only be added once therefore subsequent runs of the script would get a redefinition error. Now using the same approach as `info_disk` using simply `Add-Type`, random number is no longer needed